README.md

```diff
@@ -424,6 +424,165 @@ The local key helper is responsible for:
 - Producing signatures with the binding key
 - Cleanup of the binding key and its artifacts (when the user clears the browser session or the key is unused for a long time)
 
+### Local Key Helper on Android
```

Comment:
We also declare a list of URLs that are supported in other providers. Do we need something similar for Android? See a sample for macOS: https://github.com/alextok/dbsc-discussions/pull/4/files

Reply:
What is the use case for the list of URLs? Which components use those and how?

Reply:
Those are Idp's urls which are allowed to use the localKeyHelper.